xlive/XLive/UserManagement/CUser.cpp

```diff
@@ -427,8 +427,6 @@ void CUserManagement::ConfigureUser(XNADDR* pxna, ULONGLONG xuid, char* username
 	(*(DWORD*)&this->SecurityPacket[0]) = annoyance_factor;
 	(*(DWORD*)&this->SecurityPacket[4]) = pxna->inaOnline.s_addr;
 
-	SecurityPacket = this->SecurityPacket;
```

Comment:
why is this being removed

Reply:
Is this not assigning the global?

Reply:
No, I checked before removing to make sure it wasn't assigning to something else